tools/pipelines/test-dds-stress.yml

```diff
@@ -18,6 +18,11 @@ resources:
       - release/*
       - main
       - next
+  repositories:
```

Comment:
Adding these blocks to the pipelines that use ff_pipeline_host allows proper testing of changes to that repo before we check them in, since it means you can queue one of these pipelines while referencing a specific branch in that repository rather than being forced to use main. This is similar to the kind of thing we do for selecting the specific run of 'Build - client packages' when queueing the secondary pipelines here.